server/service/src/document/schemas/patient.json

```diff
@@ -0,0 +1,259 @@
+{
+  "$ref": "#/definitions/Patient",
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "definitions": {
+    "Address": {
+      "additionalProperties": false,
+      "properties": {
+        "address1": {
+          "description": "184097001 Patient Address",
```

Comment:
Was wondering about 18497001 ? And numbers in description for other fields ?

Reply:
These are the SNOMED identifiers. Currently just for our own documentation...